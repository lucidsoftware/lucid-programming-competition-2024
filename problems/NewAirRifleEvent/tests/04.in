200 8 7
IQzRb -10,10 -14,-12 -15,-25 6,9 20,12 0,-17 -24,19 0,-23
SEryz 25,-22 22,12 -25,-12 -20,-7 -19,24 13,2 -14,20 -2,-4
AvTIT -6,2 -19,-24 3,-16 11,-14 -16,24 7,15 9,18 -15,-8
jeCFI 5,4 20,22 11,-19 -7,3 -6,-22 6,25 -9,-7 14,-5
uoNSv -19,-23 21,17 12,-19 -25,-13 3,-10 -16,-21 -10,-16 -9,10
XxTkI -4,-17 22,17 11,-11 5,-1 2,11 16,24 4,-7 10,-5
VgOGm 15,10 -2,24 8,-23 -6,1 23,-11 -18,-22 -14,-2 24,4
dQIep 2,17 -17,22 -19,-19 16,-20 17,16 -9,19 10,-1 -25,-10
pkzAp 7,0 -18,-9 -5,-11 19,-10 -1,21 6,-13 17,17 -1,21
sWHmT 8,-14 -24,12 -9,-19 18,-23 -14,20 -3,-5 -21,2 1,10
hcZdS 18,15 -24,19 -13,-19 2,17 24,-12 -24,-13 23,-2 -3,-4
lsUEr -5,25 -10,9 9,-21 21,-14 -16,-17 23,-1 8,-4 12,4
EqcRn -24,15 0,-14 -7,14 -16,17 20,-13 22,-9 -25,3 -5,0
jVkNO -15,24 -15,-18 20,5 -4,11 -1,-10 -18,21 -16,-14 17,-11
yWztx -22,6 13,21 -21,17 -3,-6 -10,16 11,20 10,-12 10,2
HaYxu 4,18 -7,-22 -12,24 17,-6 -18,10 -9,-6 -3,1 -4,-14
vyIFI -6,-25 -5,-6 -20,-24 -25,4 16,-13 1,25 -9,20 8,19
RuKao -25,-3 14,19 7,15 7,-20 -4,13 -12,-4 8,24 -18,-24
RzTgS 18,12 22,14 18,-17 -12,17 12,20 15,-4 -15,6 14,-10
meJvN -6,-20 17,18 6,-19 -11,-11 -14,-25 3,-25 15,6 6,21
ZTjad 2,-12 -7,-25 -11,10 23,-22 -25,-15 -23,0 24,-5 -16,-13
knrcN 22,18 -2,22 4,12 6,-4 9,3 25,-3 13,13 7,1
WgGfL 19,-23 -8,0 3,18 7,-16 -5,-7 -2,-21 8,-10 13,-18
jStux -13,-13 -8,21 11,18 25,-1 20,-7 0,-9 -13,0 17,23
YEkyF 17,3 -13,-19 -10,-12 -15,-17 -8,20 -13,-8 -12,-15 -10,-21
DrVja 23,-16 -16,0 -8,10 6,22 -5,-13 -18,2 -14,2 21,23
UOsKR 3,20 -23,-6 -22,-20 11,22 -20,-24 -17,-6 -2,13 7,-22
cUjfF -22,-25 -24,-15 -10,-25 -2,-21 12,16 -7,24 4,20 0,-5
lAyPy -1,-23 22,-14 1,14 -8,-6 -16,13 -24,11 -2,-22 24,19
iSyNx -24,-19 -21,-7 -23,-21 -19,-5 -14,-3 -3,7 -14,-3 7,8
wZvtz 6,5 -18,17 5,9 18,-20 -18,23 -10,23 13,0 17,-8
GGEpB 2,-9 -12,0 7,-20 -23,-6 -2,19 22,-14 -6,-14 3,17
bpSVG 10,24 21,-10 -4,11 -18,6 8,22 -15,14 -2,16 11,-21
KeFYr 0,14 -12,-20 -14,-3 -24,-1 -9,25 -12,20 4,18 -24,-7
MOKuF 10,-12 25,3 -9,16 12,-4 19,-9 21,1 21,18 -12,5
XoiEN 4,-21 15,-18 0,-21 -19,-14 -10,-16 -5,-5 21,17 22,-10
oQdpV -22,20 8,-25 -24,5 -2,-14 8,13 -22,11 -15,-11 17,-20
yEpgN -2,15 -25,7 -12,-2 16,-7 9,3 -19,-4 -7,-11 -23,-17
Egrzg 4,25 -9,-22 -4,-13 -23,-1 -12,3 -22,-9 -15,-19 -15,11
EaegY 18,-13 19,-25 21,14 11,23 9,17 -15,23 -5,-18 -15,5
cbLEY 21,-16 -21,8 -15,17 4,-11 -19,5 -2,13 -23,-15 10,6
iuVeH 20,6 -23,-15 -9,-21 -21,-10 3,-24 -25,17 -23,-19 -14,-7
JHTeG -9,6 -15,-16 -3,-11 -21,24 13,3 -9,22 19,-15 -5,-5
JVJKA 6,10 -17,22 5,-3 11,22 -6,-25 14,-19 -16,2 -14,-25
QRZAd 14,13 10,11 -7,21 13,23 14,-25 -15,-13 6,20 -17,-5
fImRt 5,2 -11,-15 -18,25 2,24 -3,-11 -20,24 -1,6 -25,3
CjijJ 11,-13 24,22 -23,4 11,5 2,-24 0,2 4,10 18,-12
DtMIF 9,9 -17,6 -2,-16 12,-6 17,19 14,-22 -6,-15 25,19
BMJBe 24,5 21,5 -2,-1 -14,14 -9,-9 9,7 7,-14 14,22
pdfZI -22,6 -6,23 5,-5 3,8 4,9 -22,-18 -17,-10 -2,-6
iQreS -2,23 2,-2 13,12 9,15 22,25 -1,25 25,-10 18,13
qudtK -19,8 24,-5 18,17 6,15 16,10 23,1 -12,-5 24,-4
isWiy -8,-24 24,3 -21,-20 18,25 10,5 -21,-25 23,-4 4,22
XrdWC -4,-24 -10,18 -25,5 -1,-9 -5,-9 -8,20 -10,-8 9,14
SHHwp -7,-10 -22,-22 -23,18 -9,13 15,-24 19,-14 23,23 19,-7
cEwcR 23,24 14,3 8,20 12,18 -3,-9 17,5 -9,8 -2,1
RvLbZ 19,1 4,-19 -2,-3 17,-12 -19,-14 -10,-24 8,4 10,9
ryXlu 2,1 -5,-11 24,0 -1,-16 -11,-3 -7,-18 -15,-4 1,-3
gzQzN 7,-3 1,15 12,7 5,-1 23,-9 11,12 -20,24 -20,-8
pUeJM 16,-24 10,8 -25,3 1,-18 -14,-11 3,-20 -10,-23 10,22
WYFEK 2,7 4,-12 12,-17 5,0 -10,4 -16,16 -25,-6 -5,-19
pZnFs 7,13 -8,9 18,16 1,5 15,-18 5,12 7,-4 2,22
eqAKK 7,-3 17,-6 -8,19 -23,6 11,-10 -9,-23 7,-19 -9,-25
vyGui 5,-18 1,4 25,3 -11,-14 -8,-11 -17,-4 -17,-2 9,7
pQwwf -9,19 -13,10 -22,25 -6,-21 -25,15 -18,0 -22,1 -18,14
PcMJi -6,-16 13,-6 0,20 14,5 -15,-20 13,11 -20,2 -6,0
GYJcs -23,-13 -14,-3 -25,-17 19,23 3,22 25,14 22,22 1,-9
PXgKQ 15,19 -17,-17 -12,-20 -13,-3 -21,-10 -16,-9 -23,4 -21,14
sdecK 23,8 -5,-14 -23,14 -12,19 17,15 11,-17 -7,20 17,5
xKjsj 7,-11 -12,10 -6,-9 -7,13 13,-5 9,6 -9,-7 19,-9
oNbpw -15,-11 22,7 0,23 -8,24 -17,-10 -14,4 24,-22 3,-5
wZHRB -4,1 5,-22 23,0 -12,-25 -15,-15 25,-3 24,10 5,-25
lWHBm 14,-10 -24,1 23,-9 -7,20 -16,15 7,13 -13,11 16,21
baUTA 18,11 -3,24 -10,11 -4,7 -21,13 -10,25 -7,5 -1,-7
YjIUs -9,10 -23,-20 25,7 -17,-20 -24,-10 12,-22 -12,9 12,10
lhwzn -10,-17 19,-19 1,7 8,-1 -14,19 22,-7 14,-22 1,12
vurfd 1,-21 25,8 -1,-24 13,17 9,21 -8,21 -1,-24 -10,20
fAoTP -4,-23 15,13 21,12 -23,-7 -13,15 18,2 -23,12 -5,18
nLSVS 15,-17 -13,-24 6,23 16,12 -17,6 -13,17 24,24 -22,8
ewtrL -18,16 -7,-24 5,-8 -13,-12 18,0 23,10 15,-6 -5,25
qZNwv 6,13 6,-20 -5,19 -17,-8 5,-12 0,-10 1,24 -9,16
ESvEM -24,16 -17,-18 -20,-2 8,21 -7,-20 -6,0 7,23 -1,20
ZwOsw 3,-24 -11,-11 -6,17 23,-22 14,-16 5,-22 -15,-11 -15,-2
PzmCF -25,3 1,-14 15,11 -16,-5 -10,-1 -15,6 20,-15 13,-12
bkzZS 8,-18 20,-16 -7,12 0,-23 22,2 -1,13 -14,2 -22,-11
mxIlO -14,-2 -16,4 -25,-7 -2,17 -15,1 -18,-14 20,5 8,-17
VLDHN -6,12 -13,6 -13,25 25,8 16,-8 18,-2 8,11 -7,14
bdpOp 8,15 20,1 10,-19 -13,14 -23,9 -6,5 -8,-10 1,21
LfrIl 8,2 3,-18 -20,-24 8,-10 -18,-16 -6,-16 -25,25 5,-21
ysuzc 14,-16 -17,-16 24,24 1,-1 -2,15 -7,15 -5,9 15,2
vEwxO -20,11 8,-14 -21,7 -10,-10 -12,-2 -18,8 24,25 -6,17
GsuIk -19,-7 12,12 -3,-25 -21,-4 -19,25 -8,3 3,11 -7,-11
tHUvk 8,-10 11,-21 -25,-12 -4,-20 8,23 -17,5 9,6 14,-15
zgYXy -7,23 -16,-6 2,-19 24,1 19,-11 -21,10 -19,-9 -22,-1
KPTRC 13,-24 21,23 -25,-12 -6,-8 -18,-6 -7,13 24,15 -18,-21
fJTvB -1,2 -14,16 18,21 15,1 -1,-13 3,12 20,2 19,3
gzSSb -23,-12 9,11 -19,0 6,15 12,-19 -12,16 2,19 -6,3
EWaez -5,6 -12,-24 6,21 12,-14 5,-19 -3,11 -25,-12 -11,23
rdeRQ -17,13 -15,17 -6,5 5,-10 -20,-23 -14,-10 8,19 11,-8
epFbA -16,8 16,-8 12,8 -21,11 20,-21 17,-24 4,16 -5,-4
iTmhP 24,1 -21,-8 6,25 4,2 25,24 -1,10 6,2 5,2
lGpFf -7,-19 -3,17 10,9 9,-14 23,20 -23,15 -8,15 -5,-23
tMkMt -12,-10 24,21 0,-5 3,1 -23,14 22,13 8,25 17,-8
XRAma 14,-23 -19,-10 -1,13 5,3 -24,-16 -24,-9 10,14 -20,11
NlSSC 17,22 -13,17 -25,15 3,22 -17,7 12,7 11,-18 -3,18
idrTv 2,-13 22,-20 -1,2 8,10 10,7 -6,-17 -16,18 19,-12
yrcky -23,-21 7,10 7,-11 3,6 4,-14 25,11 -22,12 19,-16
IKuuX -24,16 0,-18 3,16 -23,-25 9,13 20,0 -9,17 -6,3
yZCIV -18,25 -12,4 -16,-5 5,-25 1,-21 -11,-14 7,-11 -1,21
YJxzA -24,-3 -19,17 25,-17 -19,8 -19,17 -4,8 3,-18 12,-14
zmMCk -22,-19 23,-24 13,2 -3,7 3,20 -11,3 16,24 3,-2
RCNgm 25,7 16,-1 -2,14 -24,-16 -13,-19 -24,-14 -24,-23 -18,-21
uMxws -13,-3 13,24 -15,14 15,20 1,20 8,-21 -9,-11 1,3
BHVYM 5,18 24,7 2,-5 -10,7 20,-25 -20,8 2,9 14,-3
hrKWa -11,-20 16,9 -8,-20 -14,14 12,-7 22,-1 -23,-2 -17,-15
nhiLl -25,18 8,-9 14,-19 -3,3 23,-25 -9,3 0,-11 -19,-3
dDZTg -13,-4 18,-10 -23,5 -12,5 13,13 20,-12 25,17 7,19
NiWml 14,18 -22,9 -20,-19 -13,2 9,-10 5,16 6,-9 4,2
lSxOn -3,12 3,4 16,-22 -15,7 7,13 -12,-25 20,13 8,22
ZlgRP -11,5 -23,17 7,-13 13,-17 2,-13 6,18 17,-6 -6,-12
fCAYM 4,17 -10,23 -22,20 -6,21 17,-9 -8,4 -7,-25 12,2
poytA -14,-24 5,-19 -1,22 8,-11 10,23 -20,9 -23,-14 1,5
OJXeN -1,-13 18,2 8,24 16,-25 -5,1 10,-9 11,1 8,5
mEdUl -21,21 15,4 -4,12 -19,-5 20,22 -16,-14 10,-4 24,-12
BJxeR -11,23 13,1 15,-4 13,15 12,-24 -10,4 -23,-3 -24,-12
BymJA -4,-15 -16,-20 6,-17 -18,2 13,-13 4,-2 -19,16 -11,-19
OeGnA -2,-6 12,-13 6,-15 -6,15 11,18 -22,7 -8,-24 -4,19
eZnRy -22,-5 8,5 19,-23 24,17 24,-10 -21,15 -11,22 6,13
VNsQM -15,-15 1,-3 -20,-8 -5,-6 16,-20 -24,6 -23,13 -22,6
TSUVq -4,18 -16,8 22,-22 23,-23 -7,-10 -25,-24 23,-5 -6,15
iMKpp 3,-18 10,-2 22,-15 -22,6 -23,23 2,-8 -4,22 -24,-15
CODFk -21,-25 18,-13 25,21 9,-20 19,-8 -2,-1 22,0 6,-5
NZdst 4,-23 -3,1 10,22 23,8 -4,-13 -4,-6 -9,2 -19,2
Ahuxi -15,-22 -23,19 2,8 6,0 -6,7 0,23 14,4 16,24
htjyP 8,-22 8,7 -17,18 21,24 13,18 6,14 -4,-6 2,-24
FReDD 15,-4 4,-17 -11,-22 -18,-3 2,-4 6,-8 7,-6 3,15
qdgOz 18,12 -8,15 -6,13 22,11 -5,24 21,-6 -9,19 0,3
uzPNa -24,10 1,-11 -20,18 -17,-9 2,-19 0,20 2,12 22,24
lmXrF 12,18 -20,-5 25,-1 -4,17 -18,24 -4,5 -11,-6 -9,17
CumZK 21,23 -13,-10 18,22 -2,-21 -6,18 4,-4 13,21 12,13
CldRm -24,20 12,-25 -13,-11 -12,20 20,-17 7,10 19,-20 -6,10
uAIGk 16,-22 2,11 20,-25 -8,-16 24,-7 19,-8 -22,15 -8,-19
MYGyE -1,18 3,-1 -20,-2 -10,-13 -1,-18 -19,16 -18,-11 -10,-24
otmCQ -7,-11 16,10 8,13 -22,2 20,-17 6,-22 19,-4 6,-2
DzDiu -22,22 -17,-17 -20,-11 11,11 1,0 25,-1 -14,-10 24,5
yKSTN 3,-20 -20,-24 19,5 -9,-20 25,6 -3,-23 20,-17 -16,-25
NhPLX -5,-24 -19,6 -12,15 -10,10 24,21 10,18 7,8 -17,12
peDeW 10,-13 -12,18 -15,0 1,12 -1,-5 -3,3 -25,-6 21,17
vEAbj 14,5 -25,0 0,20 -22,18 24,-11 -13,19 6,2 -19,-17
FLckC -25,17 -16,5 2,-16 5,23 8,19 3,-11 -6,-20 7,-1
FAOaN 25,17 -6,-23 20,15 -16,18 -25,-1 1,-22 -5,10 25,5
MMajj -23,-20 -22,21 4,7 19,-2 24,12 2,15 -14,21 -8,20
bzWvc 16,0 4,-25 -21,-5 -20,7 -3,6 24,-25 11,-8 -5,-11
jurPP -11,-3 13,21 4,1 21,23 -6,-25 7,-3 1,22 6,-6
rnzlD -13,12 -12,6 -15,23 6,25 -10,19 11,-5 9,7 5,0
bBbAd 7,-8 7,-12 4,-7 14,-17 13,-15 7,2 -2,-18 24,0
SVkRS -25,-4 -7,-23 3,-17 -6,9 -25,11 7,5 14,-2 -2,-24
kVVvT 1,18 1,-13 -21,-9 -10,2 15,-14 13,-24 -20,18 24,8
avKhd -23,3 -1,19 -22,2 -5,-25 8,20 -10,-19 -21,23 -7,-8
IceUq -4,-2 -13,-8 -22,9 24,-6 1,11 -25,10 -25,16 -16,17
KZuaB -7,15 8,-5 -14,-14 15,16 -6,-17 0,-16 6,-8 -2,4
XMNzH -22,-19 -5,18 0,-12 24,7 21,12 13,25 -18,8 -2,-2
qkAPq -21,-20 -21,-8 -24,-3 8,9 -6,-20 14,-19 -25,19 10,5
vkwVu 8,-14 24,22 1,-13 -3,-16 -5,-7 -1,4 -7,-2 5,-7
hkWDv -4,-23 20,-9 -13,-3 -25,6 9,-8 -12,-8 -24,-18 -17,-19
Cbtqj 16,-15 12,-11 -14,21 -19,11 0,2 -6,-9 -11,-9 21,9
ehTlQ 15,-7 -13,10 -1,-22 12,-11 4,-16 -9,21 0,15 -10,-23
hMtxp 19,-21 20,-2 -24,12 14,-21 19,16 25,-25 8,-3 -24,19
BqXBP -17,13 -1,20 14,11 -14,3 -10,-25 -25,9 -25,3 23,1
rJjeX -9,-18 -22,12 0,22 20,-23 -14,-1 -19,6 -25,2 8,-22
OCQgF 16,8 -10,-24 -14,-3 21,12 4,-1 -15,10 -16,-5 6,17
gaqWQ 23,25 -10,7 22,-12 7,22 1,-20 -22,-3 -15,16 17,1
ZRqXr -1,-5 -19,22 -4,0 19,1 -21,-23 10,17 -10,-2 1,-12
pLFAP -12,-16 -1,16 -6,-15 11,-12 -22,3 -4,6 -18,21 -15,20
VfSie -5,-8 -25,3 -17,5 -2,-15 -6,-21 -6,18 18,20 1,19
ECQOF 12,25 -13,-17 9,10 -9,22 9,2 21,14 -25,22 -15,13
bCqUX 8,6 18,-21 25,-9 -25,10 22,-4 5,-23 17,-22 -3,1
KpGOS 18,-15 9,7 -18,-6 23,20 10,-10 -2,18 -1,-10 -21,-5
rHsno 0,-19 16,-5 -22,19 -10,21 5,-13 -15,5 -20,10 6,20
kQXcc -14,-21 -18,-3 22,-7 22,-20 -6,-12 -1,-25 15,-1 -13,-16
ojxrS 20,-20 20,-13 -4,25 -11,12 -15,0 12,4 -24,-1 -11,-15
qaqTt -11,-13 20,25 9,6 -24,11 -16,-19 -2,5 2,11 -12,-18
fUVad -15,11 22,-19 -22,-8 -25,-20 25,1 12,16 23,16 4,4
tjKNQ -19,-5 3,-7 6,19 2,23 -22,19 -14,-20 24,6 8,-23
MhCDX 21,0 24,-23 20,22 -18,1 -8,8 19,25 19,17 12,7
qQXNZ 8,1 17,-20 24,-1 21,6 -24,-17 -20,23 -7,14 12,20
akbla -16,2 -1,-23 7,24 -9,-5 -9,-21 17,-4 0,-16 -7,-6
yxsiv -22,-14 25,4 1,25 6,-12 17,13 -5,3 -19,10 13,-6
juZjB 8,-22 -14,-6 13,17 16,24 -9,15 13,-21 -9,-10 15,-25
ZbSch -16,-8 18,-1 17,-9 -19,15 -11,-23 10,12 -11,5 -13,-21
vXxUp -11,-5 -15,-7 1,1 21,-13 15,-21 -14,21 5,7 25,14
CSPxq -24,21 15,-17 22,2 14,13 4,-14 -1,-15 -8,-12 -22,20
KgxHd -18,-17 2,10 -3,-9 0,19 -15,-12 -19,-18 1,-13 -18,11
sGvHW -17,6 10,22 -4,-21 -16,-2 -10,2 1,18 9,-19 2,-5
EJDnG -17,17 -18,11 3,-2 -19,21 -10,-2 -4,19 18,15 21,25
XnJFP -21,-7 -9,-8 21,17 10,14 -5,10 -8,18 2,-7 16,15
ergmb -23,18 -1,0 -23,9 -21,6 8,9 -6,-22 11,-2 -9,9
AOscL -16,-6 -19,11 8,5 -18,-20 -6,-2 10,-22 -25,-23 2,-25
Xbbku -25,0 -24,-25 -23,18 -8,20 -17,13 -14,-23 10,-24 -16,3
CTqfj -11,-8 -4,18 24,-6 -19,24 19,-19 -24,6 5,14 1,6
