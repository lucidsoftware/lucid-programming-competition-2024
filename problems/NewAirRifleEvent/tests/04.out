hcZdS
EqcRn
cUjfF
PcMJi
oNbpw
gzSSb
IKuuX
uMxws
poytA
otmCQ
KZuaB
XMNzH
bCqUX
fUVad
CTqfj
